components/resourceDetails/resourceSharing/agentAccess/index.test.jsx

```diff
@@ -20,84 +20,94 @@
  */
 
 import React from "react";
-import { mockSolidDatasetFrom } from "@inrupt/solid-client";
-import { DatasetProvider } from "@inrupt/solid-ui-react";
-import { waitFor } from "@testing-library/react";
-import userEvent from "@testing-library/user-event";
-import AgentAccess, { getDialogId, saveHandler, submitHandler } from "./index";
 
-import mockSessionContextProvider from "../../../../__testUtils/mockSessionContextProvider";
-import mockSession from "../../../../__testUtils/mockSession";
+import { fireEvent, waitFor } from "@testing-library/react";
+import userEvent from "@testing-library/user-event";
+import AgentAccess, { saveHandler } from "./index";
 
 import { renderWithTheme } from "../../../../__testUtils/withTheme";
-import { createAccessMap } from "../../../../src/solidClientHelpers/permissions";
-import useFetchProfile from "../../../../src/hooks/useFetchProfile";
 import * as profileFns from "../../../../src/solidClientHelpers/profile";
 import { mockProfileAlice } from "../../../../__testUtils/mockPersonResource";
 
 jest.mock("../../../../src/solidClientHelpers/permissions");
 jest.mock("../../../../src/hooks/useFetchProfile");
 
-const webId = "http://example.com/webId#me";
+const webId = "https://example.com/profile/card#me";
 
 describe("AgentAccess", () => {
   const permission = {
-    acl: createAccessMap(),
+    acl: {
```

Comment:
I think `createAccessMap()` is favorable to make the code easier to read (gets very noisy if you keep repeating the map)